### DISCUSSION TITLE
With new Percentage Of Billed Support how to best handle lesser of language in v1.0.0

### DISCUSSION THREAD

**Discussion by ryan-thomas-peterson** [CMSgov/price-transparency-guide#453] [Q&A] [answered]
Nearly all of our contracts that are prospectively based have language that states the rate is based on the lesser of some fraction of the billed charge (typically close to 100%) and the prospective amount that can be determined in dollars.  Based on other discussions it sounds like this may be handled using the additional information field.  When these scenarios occur can guidance be issued on what rate should be listed as primary and what should go in the additional information?

**Comment by shaselton-usds** [comment, marked as answer]
It sounds like you can go either route of reporting as a percentage and then reporting the dollar amount in the `additional_information` or reporting as a dollar negotiated amount and reporting the percentage in the `additional_information`. 

What would you think would be the best way to reflect these arrangements?

**Reply by ryan-thomas-peterson** [reply]
@shaselton-usds , this type of arrangement is not unique, so it might be good to build some alignment and I'd propose guidance that prioritizes designating a price in dollars where it's available and the 'edge cases' of 100% of billed (or whatever the 'inlier' rate is) under the additional information until further schema constructs are available.   Otherwise, a file builder might be tempted to use the 'catch all' billing_code and list 100% and then under additional_information list something more generalized about how the rate is the lesser of percent of billed and a fee schedule based on 140% of CMS RBRVS schedule (or something like that).
